parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -520,8 +555,8 @@ sync' exists serializeRaw serializeEdits b = Causal.sync exists
     writeB0 b0 = do
       for_ (view children b0) $ \c -> do
         queued <- State.get
-        when (Set.notMember (headHash c) queued)
-          $ sync' exists serializeRaw serializeEdits c
+        when (Set.notMember (headHash c) queued) $
+          sync' exists serializeRaw serializeEdits c
```

Comment:
Accidental reformatting, prob doesn't belong here... but seriously folks, who puts the dollar sign on the following line, amirite??